qt/logviewdialog.py

```diff
@@ -89,13 +89,18 @@ def __init__(self, parent, sid = None, systray = False):
         self.comboFilter.currentIndexChanged.connect(self.comboFilterChanged)
 
         self.comboFilter.addItem(_('All'), 0)
+
+        # Note about ngettext plural forms: n=102 means "Other" in Arabic and
+        # "Few" in Polish.
+        # Research in translation community indicate this as the best fit to
+        # the meaning of "all".
         self.comboFilter.addItem(' + '.join((_('Errors'), _('Changes'))), 4)
         self.comboFilter.setCurrentIndex(self.comboFilter.count() - 1)
-        self.comboFilter.addItem(_('Errors'), 1)
+        self.comboFilter.addItem(_('Errors'))
```

Comment:
I didn't dig deeper but why remove the "1" as argument if the other `addItems` do need this argument?

Reply:
I have not noticed. You are right that is a mistake. The `1` is the value of the combo box entry and has nothing to do with GNU gettext.